test/podFailover/pod-failover-workload-chart/azDiskV2-noReplica copy.yaml

```diff
@@ -0,0 +1,23 @@
+storageClass:
```

Comment:
We shouldn't have spaces in the filenames. Why is this a `copy`?

Reply:
accidentally left in. will change